src/main/webapp/app/localci/build-agents/build-agents.component.html

```diff
@@ -0,0 +1,78 @@
+<div style="padding-bottom: 60px">
+    <h3 id="build-agents-heading" jhiTranslate="artemisApp.buildAgents.title">Build Agents</h3>
+    <div class="d-flex justify-content-between align-items-center border-bottom pb-2 mb-3"></div>
+    <jhi-data-table [showPageSizeDropdown]="false" [showSearchField]="false" entityType="buildAgent" [allEntities]="buildAgents!">
+        <ng-template let-settings="settings" let-controls="controls">
+            <ngx-datatable
+                class="bootstrap"
+                [limit]="settings.limit"
+                [sortType]="settings.sortType"
+                [columnMode]="settings.columnMode"
+                [headerHeight]="settings.headerHeight"
+                [footerHeight]="settings.footerHeight"
+                [rowHeight]="settings.rowHeight"
+                [rows]="settings.rows"
+                [rowClass]="settings.rowClass"
+                [scrollbarH]="settings.scrollbarH"
+            >
+                <ngx-datatable-column prop="name" [minWidth]="150" [width]="200" [maxWidth]="200">
```

Comment:
Please dont set `minWidth`, `width` and `maxWidth`. I guess in your case you want to remove `width` to avoid a broken UI on large screens:
![image](https://github.com/ls1intum/Artemis/assets/78978542/517f9132-1f4a-4ba2-b828-f5971e1a8ba3)


same for the other columns